### ISSUE TITLE
Adjust for PrivateBin

### ISSUE THREAD

**Issue by rugk** [ziirish/zerocli#11]
Have you tested whether this works for [PrivateBin](https://github.com/PrivateBin/PrivateBin)?

If so is is okay when we add your client to our wiki?

/cc @elrido

**Comment by ziirish** [comment]
Hello,

I just gave it a quick try. Unfortunately, neither the POST nor the GET are
working.
I don't think it is hard to fix. I'll try to have a look within the next few
days.

Once it's fixed, I have no problem with you adding zerocli in your wiki.

Thanks for the interest.

**Comment by rugk** [comment]
Yes, there were many changes in PrivateBin 1.0. For instance we switched to AES GCM for encryption.
If you have any questions feel free to ask.

**Comment by rugk** [comment]
Any news here?
